packages/analytics-controller/src/AnalyticsController.test.ts

```diff
@@ -117,340 +159,447 @@ describe('AnalyticsController', () => {
+      controller.trackView('home');
 
-      expect(mockAdapter.trackPage).not.toHaveBeenCalled();
+      expect(mockAdapter.view).not.toHaveBeenCalled();
     });
   });
 
-  describe('enable', () => {
-    it('enables analytics tracking', () => {
-      const { controller } = setupController({
-        state: { enabled: false },
-      });
+  describe('optInForRegularAccount', () => {
+    it('sets optedInForRegularAccount to true', () => {
+      const { controller } = setupController();
 
-      expect(controller.state.enabled).toBe(false);
+      expect(controller.state.optedInForRegularAccount).toBe(false);
 
-      controller.enable();
+      controller.optInForRegularAccount();
 
-      expect(controller.state.enabled).toBe(true);
+      expect(controller.state.optedInForRegularAccount).toBe(true);
     });
   });
 
-  describe('disable', () => {
-    it('disables analytics tracking', () => {
+  describe('optOutForRegularAccount', () => {
+    it('sets optedInForRegularAccount to false', () => {
       const { controller } = setupController({
-        state: { enabled: true },
+        state: { optedInForRegularAccount: true },
       });
 
-      expect(controller.state.enabled).toBe(true);
+      expect(controller.state.optedInForRegularAccount).toBe(true);
 
-      controller.disable();
+      controller.optOutForRegularAccount();
 
-      expect(controller.state.enabled).toBe(false);
+      expect(controller.state.optedInForRegularAccount).toBe(false);
     });
   });
 
-  describe('optIn', () => {
-    it('opts in to analytics', () => {
+  describe('optInForSocialAccount', () => {
+    it('sets optedInForSocialAccount to true', () => {
       const { controller } = setupController();
 
-      expect(controller.state.optedIn).toBe(false);
+      expect(controller.state.optedInForSocialAccount).toBe(false);
 
-      controller.optIn();
+      controller.optInForSocialAccount();
 
-      expect(controller.state.optedIn).toBe(true);
+      expect(controller.state.optedInForSocialAccount).toBe(true);
     });
   });
 
-  describe('optOut', () => {
-    it('opts out of analytics', () => {
+  describe('optOutForSocialAccount', () => {
+    it('sets optedInForSocialAccount to false', () => {
       const { controller } = setupController({
-        state: { optedIn: true },
+        state: { optedInForSocialAccount: true },
       });
 
-      expect(controller.state.optedIn).toBe(true);
+      expect(controller.state.optedInForSocialAccount).toBe(true);
 
-      controller.optOut();
+      controller.optOutForSocialAccount();
 
-      expect(controller.state.optedIn).toBe(false);
+      expect(controller.state.optedInForSocialAccount).toBe(false);
     });
   });
 
-  describe('messenger actions', () => {
-    it('handles trackEvent action', () => {
-      const mockAdapter = createMockAdapter();
-      const { messenger } = setupController({
-        platformAdapter: mockAdapter,
+  describe('selectAnalyticsId', () => {
```

Comment:
Should the tests for selectors be in this file? It seems like we already have tests for them in `selectors.test.ts`.